recipe/meta.yaml

```diff
@@ -32,8 +32,7 @@ requirements:
 test:
   requires:
     - python <3.10
-  imports:
-    - certifi
+    - pip
 
 about:
   home: http://certifi.io/
```

Comment:
The license should be `MPL-2.0`, see https://github.com/certifi/python-certifi/blob/master/LICENSE

Reply:
okay sure and does this url needs to be updated  in the doc_source_url ? @skupr-anaconda

Reply:
No, it's an upstream license file. We need update only `license: MPL-2.0`